tools/pkglint/.eslintrc.js

```diff
@@ -1,3 +1,202 @@
-const baseConfig = require('cdk-build-tools/config/eslintrc');
-baseConfig.parserOptions.project = __dirname + '/tsconfig.json';
-module.exports = baseConfig;
+// This cannot reference the build rules from cdk-build-tools as this
+// package is itself used by cdk-build-tools.
```

Comment:
So this is just a copy/paste of the file in `cdk-build-tools`? Does that mean we now need to keep these two files in sync? One option would be to change everything to reference this file instead (e.g., `require('pkglint/config/eslintrc')`), which is a fairly broad change I guess. The other would be to at the very least add a comment at the top of the cdk-build-tools version stating that the two need to be kept in sync. What do you think?

Reply:
Since `pkglint` doesn't get that many changes, and since it's not really "production code", I'm comfortable letting them drift.

Reply:
You know what, `awslint` needs the same treatment :(

Reply:
Are you updating this PR with the `awslint` changes then, or opening a separate one?

Reply:
Could we do a copy as part of the 'build' npm script?